urth_dash_js/notebook/dashboard-view/dashboard.js

```diff
@@ -37,6 +37,9 @@ define([
     var cssLoaded = false;
     var idCounter = 0;
 
+    // duration => the resize transition on gridstack and gridstack cells
+    var RESIZE_DURATION = 350;
```

Comment:
This is necessary due to [this bug](https://github.com/troolee/gridstack.js/issues/159) in gridstack where a `resizestop` event is fired while a CSS transition is taking place. 

The value is not hardcoded in the gridstack code; rather it is defined in `gridstack.css` as a `transition-duration`. There is a 300ms CSS transition on cell width/height that we are waiting to complete before taking action based on the size of a cell after resizing.